libpod/oci.go

```diff
@@ -215,7 +215,7 @@ func (r *OCIRuntime) createContainer(ctr *Container, cgroupParent string) (err e
 		cmd.Env = append(cmd.Env, fmt.Sprintf("NOTIFY_SOCKET=%s", notify))
 	}
 	if listenfds, ok := os.LookupEnv("LISTEN_FDS"); ok {
-		cmd.Env = append(cmd.Env, fmt.Sprintf("LISTEN_FDS=%s", listenfds))
+		cmd.Env = append(cmd.Env, fmt.Sprintf("LISTEN_FDS=%s", listenfds), fmt.Sprintf("LISTEN_PID=1"))
```

Comment:
Does that last part need to be an Sprintf()? Can't it just be a string?

Reply:
Yup.